esp/platform/espcontext.cpp

```diff
@@ -820,6 +820,22 @@ static IEspContainer*& getContainer()
     return gContainer;
 }
 
+LogRequest readLogRequest(char const* req)
+{
+    if (isEmptyString(req))
```

Comment:
should probably treat old values of logRequests=false and logRequests=0 as LogRequestsNever.